renovate.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ]
+}
```

This is what we use in other policy repos
```suggestion
{
  "extends": [
    "config:base",
    "group:allNonMajor",
    "schedule:earlyMondays"
  ],
  "labels": ["dependencies"]
}
}
```